docs/usage/examples/self-hosting.md

```diff
@@ -396,3 +396,35 @@ The logging level output is controlled by the Bunyan logging library.
 |    40 | warn    |
 |    50 | error   |
 |    60 | fatal   |
+
+## Self-signed TLS/SSL certificates
+
+Renovate and invoked helper programs (e.g. Git, npm) use a secure TLS connection (e.g. HTTPS) to connect to source code platforms and dependency hosts.
+If the systems use any self-signed certificates or certificate authorities then Renovate needs to be configured to trust these additional certificates.
```

Comment:
It's not clear to me what we mean when we say `If the systems use any self-signed certificates`? Do we mean the `Renovate and invoked helper programs (e.g. Git, npm)`?

Reply:
More the remote systems ("source code platforms and dependency hosts"). Maybe "remote hosts" is more clear?

```suggestion
If the remote hosts use any self-signed certificates or certificate authorities then Renovate needs to be configured to trust these additional certificates.
```

Overall both Renovate and all invoked helper programs, which interact with remote hosts, need to be told about a self-signed certificate.

Reply:
resolved via f3472ec